web_service/xmlrpc_test.py

```diff
@@ -0,0 +1,34 @@
+import xmlrpc.client
+import functools
+
+
+SERVER = "localhost"
+PORT = 8069
+DB_NAME = "practice"
+USER = "admin"
+PASSWORD = "admin"
+
+ROOT = 'http://%s:%d/xmlrpc/' % (SERVER, PORT)
+uid = xmlrpc.client.ServerProxy(ROOT + 'common').login(DB_NAME, USER, PASSWORD)
+sock = functools.partial(xmlrpc.client.ServerProxy(ROOT + 'object').execute, DB_NAME, uid, PASSWORD)
+
+model = "open.academy.session"
+method = "search_read"
+domain = []
+
+list_session_before = sock(model,method,domain,["name","number_seats"])
```

Comment:
You need to add space after comma, this is a lint.
```suggestion
list_session_before = sock(model, method, domain, ["name","number_seats"])
```